frontend/event/eventController.js

```diff
@@ -95,6 +95,7 @@
             promise.then(function success() {
                 eventCtrl.events = eventCtrl.events.filter(thisEvent => thisEvent.key  !== event.key);
                 MessageService.showToast('Evento removido com sucesso!');
+                if (eventCtrl.isEventPage) $state.go('app.user.events');
```

Comment:
This verification is very confusing, why you have a flag to ensure a state that should be linked directly to a controller?